### PR TITLE
Updated to spring boot 2

### PR DESCRIPTION
**Pull request by rverma-jm**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@
 language: java
 sudo: false
 jdk:
-- openjdk8
+- openjdk13
```
Comment (aasaru) [inline .travis.yml:22]: Why you want to change that?

File: build.gradle [hunk 2]
```diff
@@ -15,44 +15,32 @@
...
 
 group 'org.apache.fineract.cn'
-version '0.1.0-BUILD-SNAPSHOT'
+version 'spring_boot_2-SNAPSHOT'
 
 tasks.withType(JavaCompile) {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
+    sourceCompatibility = JavaVersion.VERSION_13
+    targetCompatibility = JavaVersion.VERSION_13
```
Comment (aasaru) [inline build.gradle:43]: Why you want to upgrade Java version?
Anyone currently using Fineract-CN is running it on Java8 (as Spring Boot 1.x doesn't work above Java 8). Why you want them to make two big changes at the same time. Let's only upgrade Spring & Spring Boot in this iteration and let's leave upgrading everything else to the future.

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-bin.zip
```
Comment (aasaru) [inline gradle/wrapper/gradle-wrapper.properties:3]: Are you sure you want to upgrade Gradle verison as well? You can do that but the best pracise is to make these kind of breaking changes incrementally.

**Review by aasaru — CHANGES_REQUESTED** [review]
Please stick to Java 8 for now and let's do update of Java version in some future iteration.

File: gradle/wrapper/gradle-wrapper.properties (hunk 3 above)
Comment (rverma-jm) [reply gradle/wrapper/gradle-wrapper.properties:3]: we are using openapi generator plugin which is kind of not compatible with gradle 4.10, and even spring boot plugin did have some issues with gradle version.

**Review by aasaru — COMMENTED** [review]
Also to the beginning of Git commit comment, please add JIRA ticket ID.
FINCN-172 
in current case. This way we can better link code with tasks. JIRA also then automatically picks up all the code changes (we have a lot of repositories as you know - it would be easy to miss some PR).

File: build.gradle (hunk 2 above)
Comment (rverma-jm) [reply build.gradle:43]: basically java 8 is not supported by oracle, I pretty much believe whoever is using java8 is not by choice. I can downgrade to java 11 which is LTS but so far I see no point in sticking to java8. 
It was a very clear decision within our team as well that no java services below 11 would be developed.

Again fineract-cn is not being installed as java software, its a container service, until and unless container is running fine, its an easy task to replace.  We are packaging runtime, thats the whole point of containerization.

File: gradle/wrapper/gradle-wrapper.properties (hunk 3 above)
Comment (aasaru) [reply gradle/wrapper/gradle-wrapper.properties:3]: Ok, I see no harm in upgrading Gradle version.

File: build.gradle (hunk 2 above)
Comment (aasaru) [reply build.gradle:43]: OK, let's stick to Java 11 then. Please downgrade to Java 11.

**Pull request closed without merging**